README.md

```diff
@@ -15,27 +11,33 @@ Visit [firepad.io](http://demo.firepad.io/) to see a live demo of Firepad in ric
 [![a screenshot of demo.firepad.io including a picture of two cats and a discussion about fonts](screenshot.png)](http://demo.firepad.io/)
 
 ## Setup
-Firepad uses [Firebase](https://firebase.google.com/?utm_source=firepad) as a backend, so it requires no server-side
+Firepad uses [Firebase](https://firebase.google.com) as a backend, so it requires no server-side
```

Comment:
Google Analytics automatically uses it.  It's low-pri, but I'll probably add it back.